### DISCUSSION TITLE
model lost texture

### DISCUSSION THREAD

**Discussion by dungnguyen234** [google/model-viewer#4227] [Q&A]
i have use this scan model for display in AR with model viewer , the problem is that this model have texture when i open with https://3dviewer.net/ but can not when using with model viewer , this is file model :

https://drive.google.com/file/d/1zPMWu_nZZzlVGaBPMS_pSfyWTdz4wGUG/view?usp=share_link

please help me to figure out this issue

**Comment by elalish** [comment]
Your file uses the SpecularGlossiness extension, which is deprecated (hence no color). Our editor *should* be able to automatically update it for you, but there's a bug blocking that: https://github.com/donmccurdy/glTF-Transform/issues/916

You should be able to use the `gltf-transform` CLI to do the conversion for you as well.
